Source/Laser/LaserProfilesImpl/LaserProfileFromFile.cpp

```diff
@@ -178,32 +180,46 @@ WarpXLaserProfiles::FromFileLaserProfile::parse_lasy_file(std::string lasy_file_
 
         WARPX_ALWAYS_ASSERT_WITH_MESSAGE(E.getAttribute("dataOrder").get<std::string>() == "C",
                                          "Reading from files with non-C dataOrder is not implemented");
-        auto axisLabels = E.getAttribute("axisLabels").get<std::vector<std::string>>();
-        auto fileGeom = E.getAttribute("geometry").get<std::string>();
-        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(fileGeom == "cartesian", "WarpX can only read laser files with cartesian 3D geometry.");
-        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(axisLabels[0] == "t" && axisLabels[1] == "y" && axisLabels[2] == "x",
-                                         "WarpX expects laser files with axisLabels {t, y, x}.");
 
+        m_params.fileGeom = E.getAttribute("geometry").get<std::string>();
         auto E_laser = E[io::RecordComponent::SCALAR];
         auto extent = E_laser.getExtent();
-        //Dimensions of lasy file datas: {t,y,x}
-        m_params.nt = extent[0];
-        m_params.ny = extent[1];
-        m_params.nx = extent[2];
-        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(m_params.nt > 1, "nt in lasy file must be >=2");
-        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(m_params.nx > 1, "nx in lasy file must be >=2");
-        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(m_params.ny > 1, "ny in lasy file must be >=2 in 3D");
         // Extract grid offset and grid spacing
         std::vector<double> offset = E.gridGlobalOffset();
         std::vector<double> position = E_laser.position<double>();
         std::vector<double> spacing = E.gridSpacing<double>();
-        // Calculate the min and max of the grid
-        m_params.t_min = offset[0] + position[0]*spacing[0];
-        m_params.t_max = m_params.t_min + (m_params.nt-1)*spacing[0];
-        m_params.y_min = offset[1] + position[1]*spacing[1];
-        m_params.y_max = m_params.y_min + (m_params.ny-1)*spacing[1];
-        m_params.x_min = offset[2] + position[2]*spacing[2];
-        m_params.x_max = m_params.x_min + (m_params.nx-1)*spacing[2];
+        if (m_params.fileGeom=="thetaMode") {
+            //Dimensions of lasy file data: {m,t,r}
+            amrex::Print() << Utils::TextMsg::Info( "Found lasy file in RZ geometry" );
+            m_params.n_rz_azimuthal_components = extent[0];
+            m_params.nt = extent[1];
+            m_params.nr = extent[2];
+            if(m_params.nt <= 1) Abort("nt in lasy file must be >=2");
+            if(m_params.nr <= 1) Abort("nr in lasy file must be >=2");
```

Comment:
Just a quick suggestion:
```suggestion
            if(m_params.nt <= 1) WARPX_ABORT_WITH_MESSAGE("nt in lasy file must be >=2");
            if(m_params.nr <= 1) WARPX_ABORT_WITH_MESSAGE("nr in lasy file must be >=2");
```